core/roboconf-dm/src/test/java/net/roboconf/dm/internal/test/TestManagerWrapper.java

```diff
@@ -101,6 +104,32 @@ public IMessagingClient getInternalMessagingClient() throws IllegalAccessExcepti
 
 
 	/**
+	 * @param ma a managed application with a non-null directory
```

Comment:
All these methods have no description.
They are test utilities, we do not have the same expectations than for **main** sources. But you are right, it could be nice to document all these functions (not just the two new ones). This is not an emegency IMO.